packages/cdk/.npmrc

```diff
@@ -0,0 +1,2 @@
+engine-strict=true
```

Comment:
I think this will cause more problems than it's worth. We don't have strict NPM version requirements at the moment outside of a minimum version of 14. If a package is untested for a new version of NPM that the customer is on, then they are effectively blocked from using AGC on that version.

If you're just looking to prevent accidental lockfile version upgrades, please use [lockfile-version](https://docs.npmjs.com/cli/v8/using-npm/config#lockfile-version) instead.

Reply:
makes sense, i change rules to strictly allow only version >=14.0.0 (which comes with npm >=6.0.0), then use lockfile-version flag

Reply:
Update: removing any engine strict rules and using lockfile-version flag only